app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: 'sign-in-form' }) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'form_field' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password", class: 'form_field' %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in", class: 'form_field' %>
+  </div>
+<% end %>
+
+<div class="messages">
+<<<<<<< Updated upstream
+  <%= render "devise/shared/links" %>
+=======
+  <%= render "devise/shared/links" %>
+>>>>>>> Stashed changes
+</div>
```

Comment:
- [ ] When I run your app I get an error and I can't check if a user can log in or signup, please kindly fix the error and make sure your devise is setup correctly 😁 
- [ ] As you can see there are two ` <%= render "devise/shared/links" %>` you should accept the current change, and there is a space at the end of `...links" %>` it should be `...links"%>`. Kindly fix the space error in the `registrations/new.html.erb` file and everything will be fine 😁 

<img width="793" alt="Screenshot 2023-05-24 at 22 19 43" src="https://github.com/chelobotix/Blog-Ruby-on-Rails/assets/74501523/3200a59e-135f-403c-b2bd-037e74a275b9">

Reply:
@cynthiainga thank you for your observations. I didn't notice that duplicate link